quail/processes/wps_climdex_quantile.py

```diff
@@ -100,6 +99,20 @@ def collect_args_wrapper(self, request):
 
         return [data_file] + literal_inputs
 
+    def unpack_data_file(self, data_file, data_vector):
+        try:
+            return load_rdata_to_python(data_file, data_vector)
+        except (RRuntimeError, ProcessError, IndexError):
+            pass
```

Comment:
I would suggest adding a comment or a docstring explaining why we pass on this error.